packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -221,6 +221,19 @@ modifiers in the camelCase format: `Dropdown--top`, `Dropdown--rightTop`, `Dropd
 Current class combinations (`Dropdown--top.Dropdown--left`, `.Dropdown--top.Dropdown--right`,
 `.Dropdown--bottom.Dropdown--left`, `.Dropdown--bottom.Dropdown--right`) will be removed in the next major release.
 
+## Feature Flag: Enhanced Shadow
+
+The enhanced shadow feature is disabled by default. To enable it, you have two options:
+
+1. To enable it globally, set the `$enable-dropdown-enhanced-shadow` feature flag to `true`. See main
+   [README][readme-feature-flags] for more info.
+2. Another option is to use the `spirit-enable-dropdown-enhanced-shadow` CSS class on any parent of the
```

Comment:
What about documenting this in the main readme? E.g. that the functionality will always be supported by `_featureFlags.scss` and by class on element. So you can mention here only the name of the feature flag and then link it to the main readme to usage.